numpy/random/pcg64.pyx

```diff
@@ -45,33 +35,18 @@ cdef double pcg64_double(void* st) nogil:
     return uint64_to_double(pcg64_next64(<pcg64_state *>st))
 
 
-cdef class PCG64:
-    u"""
-    PCG64(seed=None, inc=0)
+cdef class PCG64(BitGenerator):
```

Comment:
Not related to this specifically, but I think

https://github.com/numpy/numpy/blob/03069a156103c6db175afa9112d0cfebe882d91b/numpy/random/src/pcg64/pcg64.h#L175

has a bug.  Shouldn't it be 0x8000000000000000ULL not 0x800000000000 to get the top bit?

I don't think the tests do not push this value out of the lower 62 bits  so it doesn't catch.

Reply:
Yeah, probably needs fixing. Keep it in mind.

Reply:
Ugh. Yuck. Must have miscounted the 0s. Even with the right constant, it's wrong. The RHS should just be `initseq.low >> 63u`, I think. We want a result of `0` or `1` here, not `0` or `0x8000000000000000ULL`

Reply:
Just to be sure, just `initseq.low >> 63u`

Reply:
Tests are passing now, so once this is fixed we can put it in and review for nits without working with a mega set of files.

Reply:
see #13837